frontend/packages/topology/src/filters/TopologyFilterBar.tsx

```diff
@@ -78,7 +80,12 @@ const TopologyFilterBar: React.FC<TopologyFilterBarProps> = ({
     <Toolbar className="co-namespace-bar odc-topology-filter-bar">
       <ToolbarContent>
         <ToolbarItem>
-          <QuickSearchButton onClick={() => setIsQuickSearchOpen(true)} />
+          <QuickSearchButton
+            onClick={() => {
+              fireTelemetryEvent('Quick Search Accessed', null);
```

Comment:
You can omit the `null` parameter if you make it optional.

Also, you should move this to `TopologyView` as mentioned in another comment.